lib/tasks/fake_data.rake

```diff
@@ -109,38 +112,43 @@ namespace :fake_data do
 
   desc 'create assessment questions'
   task create_assessment_questions: :environment do
+    puts 'create_assessment_questions...'
     ASSESSMENT_QUESTIONS.each do |attribute_values|
       AssessmentQuestion.create!(attribute_values)
     end
   end
 
   desc 'create genders'
   task create_genders: :environment do
+    puts 'create_genders...'
     GENDERS.each do |gender|
       Gender.create!(key: gender.parameterize(separator: '_'), title: gender)
     end
   end
 
   desc 'create ethnicities'
   task create_ethnicities: :environment do
+    puts 'create_ethnicities...'
     ETHNICITIES.each do |attributes|
       Ethnicity.create!(attributes)
     end
   end
 
   desc 'create identifier types'
   task create_identifier_types: :environment do
+    puts 'create_identifier_types...'
     IDENTIFIER_TYPES.each do |attributes|
       IdentifierType.create!(attributes)
     end
   end
 
   desc 'create fake moves'
   task create_moves: :environment do
+    puts 'create_moves...'
     people = Person.all
     prisons = Location.where(location_type: 'prison').all
     courts = Location.where(location_type: 'court').all
-    20000.times do
+    1000.times do
```

Comment:
rather than fighting over the value of this, could it not be a parameter?

Reply:
I see your point, but the effort is not worth,
it is not trivial, since this task is called from another tasks
we need to change the 2 tasks, making things more complicated for something that probably nobody will use.
I'll be happy to do the change if there is a real need.